Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -196,3 +196,21 @@ extension Event.EventName {
         }
     }
 }
+
+struct ErrorResponse: Codable {
+    let errors: [ErrorDetail]
+}
+
+struct ErrorDetail: Codable {
+    let id: String
+    let status: Int
+    let code: String
+    let title: String
+    let detail: String
+    let source: ErrorSource
+    let meta: [String: String]
```

Comment:
Is the meta always this shape? Also if we aren't reading maybe better to leave it out until we need really it.

Reply:
Good call, removed this since we don't use it.